entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -3,40 +3,40 @@ title: EF Core tools reference (.NET CLI) - EF Core
 description: Reference guide for the Entity Framework Core .NET Core CLI tools
 author: bricelam
 ms.author: bricelam
-ms.date: 09/09/2020
+ms.date: 09/17/2020
 uid: core/miscellaneous/cli/dotnet
 ---
 
 # Entity Framework Core tools reference - .NET Core CLI
 
 The command-line interface (CLI) tools for Entity Framework Core perform design-time development tasks. For example, they create [migrations](/aspnet/core/data/ef-mvc/migrations), apply migrations, and generate code for a model based on an existing database. The commands are an extension to the cross-platform [dotnet](/dotnet/core/tools) command, which is part of the [.NET Core SDK](https://www.microsoft.com/net/core). These tools work with .NET Core projects.
 
-If you're using Visual Studio, we recommend the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) instead:
+When using Visual Studio, consider using the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell) in lieu of the CLI tools. Package Manager Console tools automatically:
 
-* They automatically work with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
-* They automatically open files generated by a command after the command is completed.
+* Works with the current project selected in the **Package Manager Console** without requiring that you manually switch directories.
+* Opens files generated by a command after the command is completed.
```

Comment:
You also get tab completion of commands, parameters, project names, context types, and migration names.